src/Exception/ForcedException.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace Robo\Exception;
+
+class ForcedException extends \Exception
```

Comment:
Naming things is hard. "Forced" exception seems unclear to me. "AbortRollbackException" would be more clear, except that this might also happen in a completion function. It's also possible to use `ignoreErrorsTaskWrapper` for a purpose completely unrelated to rollback / completion (although I don't know that this is very common).

How about `AbortTasksException`?

A comment in this class file to complement the documentation already existing elsewhere (in this PR) would also be helpful.

Reply:
@greg-1-anderson that works for me! Updated.